### PR TITLE
changed file_exists to file to allow urls

### LINKED ISSUE #55: Publicly accessible URLs throw exception
**Problem**

FFmpeg is capable of getting video information through (publicly accessible) URLs. Currently both the `FFProbeProvider` and `FFMpegProvider` include a `!file_exists` to check if a file exists on the current file system.

**Suggestion**

Change `file_exists` to `file`.

### PR DESCRIPTION
**Pull request by ToshY**
Fixes #55

### CODE REVIEW

File: src/OutputProviders/FFMpegProvider.php [hunk 1]
```diff
@@ -34,7 +34,7 @@ public function getOutput()
         }
 
         // File doesn't exist
-        if (!file_exists($this->movieFile)) {
+        if (@file_get_contents($this->movieFile, false, null, 0, 1) === false) {
```
Comment (char0n) [inline src/OutputProviders/FFMpegProvider.php:37]: `0/0` as `offset/length` is supported as well. Let's use this one as it allows to process empty files as well.
```suggestion
        if (@file_get_contents($this->movieFile, false, null, 0, 0) === false) {
```

File: src/OutputProviders/FFProbeProvider.php [hunk 2]
```diff
@@ -34,7 +34,7 @@ public function getOutput()
         }
 
         // File doesn't exist
-        if (!file_exists($this->movieFile)) {
+        if (@file_get_contents($this->movieFile, false, null, 0, 1) === false) {
```
Comment (char0n) [inline src/OutputProviders/FFProbeProvider.php:37]: `0/0` as `offset/length` is supported as well. Let's use this one as it allows to process empty files as well.
```suggestion
        if (@file_get_contents($this->movieFile, false, null, 0, 0) === false) {
```

**Review by char0n — CHANGES_REQUESTED** [review]
Thanks for contributing! Please have a look at my code review comments. Also if we're introducing this feature, it would be good to add tests for it. Please use this testing file for testing: https://github.com/char0n/ffmpeg-php/blob/master/tests/data/test.mp4?raw=true. I can imagine following tests:

- testing if FFMpegProvider can handle URL
- testing if FFProbeProvider can handle URL
- testing if Movie can handle URL
- testing what happens when provided URL cannot be fetched

**Comment by ToshY** [comment]
Well the CI seems to be fine with it, but I honestly don't know if it's okay like this, as I normally don't write tests. So I just tried to mimic the other tests and ended up with this. 
Let me know if there's something that needs to be changed.

**Review by char0n — APPROVED** [review]
The tests satisfy the requirements. Nicely done. I'll just do some minor cleanup in them before the release.

**Pull request merged**